### DISCUSSION TITLE
Async update of config.json

### DISCUSSION THREAD

**Discussion by ghost** [deepch/RTSPtoWSMP4f#28] [Ideas]
Hi, 
is it possible after I insert new RTSP link in config.json that software reads it? Maybe some interval or simular? I need to add, delete and edit config.json without stopping and building GO binaries. 

Huge respect for you work!

**Comment by deepch** [comment]
try https://github.com/deepch/RTSPtoWeb and use http api

**Comment by xiazeyang-evan** [comment]
it is support  record the vedio in server ?
